test/unit/specs/Todo.spec.ts

```diff
@@ -0,0 +1,16 @@
+import Todo from "src/components/Todo";
+import { assert } from "chai";
+
+describe("Todo.vue", () => {
+  it("renders currect contents", () => {
+    const vm = new Todo({
+      propsData: {
+        todoId: 1, description: "Todo 1", type: "ROOT",
+        estimatedSize: 2, status: false
+      }
+    }).$mount();
+
+    const actual = vm.$el.textContent;
+    assert.match(actual, /Todo 1/);
```

Comment:
OTL I missed it. I misunderstood it's for collection
http://chaijs.com/api/assert/#method_include